test/basic.jl

```diff
@@ -240,3 +241,29 @@ end
     dict = build_project_dict("Clang", v"9.0.1+2", dependencies)
     @test dict["compat"]["libLLVM_jll"] == "8.3"
 end
+
+
+@testset "valid_library_path" begin
```

Comment:
Why not testing in [`test/auditing.jl`](https://github.com/JuliaPackaging/BinaryBuilder.jl/blob/d0437c4ed649fa335ac825f04d5e2410ea1f04c7/test/auditing.jl)?

Reply:
I actually looked at `test/auditing.jl` first, but didn't see where to fit it in. I am completely neutral on this, though. If you tell me where you want it, I'll move it there

Reply:
It's just that this is "audit" stuff anyway (and so you don't need to specifically load `Auditor` at the beginning), I don't think this function is used anywhere else (but I din't actually check this claim)

Reply:
moved it